src/main/java/mate/jdbc/service/impl/ManufacturerServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.ManufacturerService;
+
+@Service
+public class ManufacturerServiceImpl implements ManufacturerService {
+    @Inject
+    private ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        return manufacturerDao.get(id);
+    }
```

Comment:
On the service layer, you should return the right object(Manufacturer) or throw an exception